bindings/wasm/CHANGELOG.md

```diff
@@ -19,6 +19,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.2 - 2023-MM-DD
+
+Same changes as https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/CHANGELOG.md.
+
+### Fixed
+
+- Runtime serializtion error of `BigInt` when working with native token amounts;
```

Comment:
Not exactly sure what's best, but technically this hasn't been done in this crate anymore

Reply:
But it was broken in Wasm and now it's fixed 🤷🏻

Reply:
Yeah but I mean, not because of a change in the wasm bindings, let me think about it

Reply:
Technically it is, it was copied from somewhere else, but it's a part of the Wasm bindings.

Reply:
Alright 👍🏻